### ISSUE TITLE
region changes latitude node values

### ISSUE THREAD

**Issue by anderson276** [CDAT/cdms#136]
```python
regionNA = cdutil.region.domain(latitude=(-50.,50.,'ccb'))
```

input latitude:
```
[-90. -86. -82. -78. -74. -70. -66. -62. -58. -54. -50. -46. -42. -38. -34.
 -30. -26. -22. -18. -14. -10.  -6.  -2.   2.   6.  10.  14.  18.  22.  26.
  30.  34.  38.  42.  46.  50.  54.  58.  62.  66.  70.  74.  78.  82.  86.
  90.]
```
bounds
```
[[-90. -88.]
 [-88. -84.]
 [-84. -80.]
 [-80. -76.]
 [-76. -72.]
 [-72. -68.]
 [-68. -64.]
 [-64. -60.]
 [-60. -56.]
 [-56. -52.]
 [-52. -48.]
 [-48. -44.]
 [-44. -40.]
 [-40. -36.]
 [-36. -32.]
 [-32. -28.]
 [-28. -24.]
 [-24. -20.]
 [-20. -16.]
 [-16. -12.]
 [-12.  -8.]
 [ -8.  -4.]
 [ -4.   0.]
 [  0.   4.]
 [  4.   8.]
 [  8.  12.]
 [ 12.  16.]
 [ 16.  20.]
 [ 20.  24.]
 [ 24.  28.]
 [ 28.  32.]
 [ 32.  36.]
 [ 36.  40.]
 [ 40.  44.]
 [ 44.  48.]
 [ 48.  52.]
 [ 52.  56.]
 [ 56.  60.]
 [ 60.  64.]
 [ 64.  68.]
 [ 68.  72.]
 [ 72.  76.]
 [ 76.  80.]
 [ 80.  84.]
 [ 84.  88.]
 [ 88.  90.]]

```

output:
```
[-49. -46. -42. -38. -34. -30. -26. -22. -18. -14. -10.  -6.  -2.   2.   6.
  10.  14.  18.  22.  26.  30.  34.  38.  42.  46.  49.]
```

**Comment by dnadeau4** [comment]
The documentation says:
>The data extracted above will have the bounds and weights of the region extracted precisely. Some commonly used domains have been pre-defined for convenience. They are:
https://uvcdat.llnl.gov/documentation/utilities/utilities-1.html

@doutriaux1  From the [cdutil code](https://github.com/UV-CDAT/cdutil/blob/master/Lib/region.py#L184-L212), it seems that you have to specify the `bounds` and not the `center latitudes` which can be confusing. 

@anderson276 It works with the following line (using `bounds` instead of [-50, 50] `center latitudes` values).

```python
regionNA = cdutil.region.domain(latitude=(-52.,52.,'ccb'))
```

(BTW,  `ccb` is overwritten with `oob`) 
See:
https://github.com/UV-CDAT/cdutil/blob/master/Lib/region.py#L87
https://github.com/UV-CDAT/cdutil/blob/master/Lib/region.py#L142
```
[-50. -46. -42. -38. -34. -30. -26. -22. -18. -14. -10.  -6.  -2.   2.   6.
  10.  14.  18.  22.  26.  30.  34.  38.  42.  46.  50.]
```

**Comment by doutriaux1** [comment]
@dnadeau4 yes you pass the edges of the region of interest and we reset the bounds to match exactly this region. And you're right there should be no 3rd argument, I forgot about this.. Can you please tweak it to return an error if the user passes a 3rd argument.

**Comment by dnadeau4** [comment]
@anderson276 was asking if you regrid the data when you change the bounds, since it changes the area of that cell.

**Comment by doutriaux1** [comment]
no the data is not changed. It is just a selector, the whole idea of this is to be used to be passed as an input to either an averager or a regridder. If you are interested in say Northern Hemisphere but your model's ltitudes expand from -5 to 5. You will want to `weight` the `0 latitude` cells by only an half weight since half of the data is in the Southern Hemisphere. By resetting the bounds, other tools (such as cdutil.averager, regridder, etc..) will **THEN** generate appropriate weights

**Comment by dnadeau4** [comment]
@anderson276 Is this what you want.  That makes sense to me.

**Comment by doutriaux1** [comment]
@dnadeau4 we just need to makesure we can't pass the third argument. Do you mind putting that in? Thanks.

**Comment by dnadeau4** [comment]
@anderson276 can we close this issue? Thanks!

**Comment by anderson276** [comment]
Yes you can, thanks!

**Comment by doutriaux1** [comment]
@dnadeau4 did you add the check so that we cannot pass the 3rd argument?
